### PR TITLE
feat(data-explorer): added bootstrap logic to add settings if they ar…

### PR DESCRIPTION
**Pull request by jelmerveen**
…e not present.

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] Conventional commits (squash if needed)
- [x] No warnings during install
- [x] Updated javascript typing

### CODE REVIEW

**Comment by jelmerveen** [comment]
Test: remove app settings package from master dev
navigator -> appsettings  -> check the box -> red button

start application without admin, red toast should appear

login, try again, green toast should appear.

Go back to navigator on master dev, check that settings are present.

File: packages/data-explorer/src/main.ts [hunk 1]
```diff
@@ -19,6 +20,10 @@ router.beforeEach((to, from, next) => {
 // This way the mainView can coordinate the events
 Vue.prototype.$eventBus = new Vue()
 
+BootstrapExplorer().then((statusMessage: string | undefined) => {
+  if (statusMessage) store.dispatch('notifyUser', { type: 'success', message: statusMessage })
```
Comment (MaxPostema) [inline packages/data-explorer/src/main.ts:24]: Wellicht een warning gebruiken?

Comment (MaxPostema) [inline packages/data-explorer/src/main.ts:24]: Wellicht is dit een warning?
'Succes' nodigt mij niet uit om als admin in te loggen.

Comment (MaxPostema) [inline packages/data-explorer/src/main.ts:24]: Suggest to use warning here? Success is not triggering me to login as admin.

**Pull request merged**